dotCMS/src/integration-test/java/com/dotcms/publisher/bundle/business/BundleFactoryTest.java

```diff
@@ -0,0 +1,215 @@
+            assertTrue(bundlesSentIds.contains(bundleId));
+        }
+
+        deletePublishingBundle(bundleIdsAdmin);
+        deletePublishingBundle(bundleIdsUser);
+    }
+
+    @Test
+    public void test_findSentBundlesOlderThan_byAdminUser() throws DotDataException {
+        //Create a few bundles that the owner is the Admin
+        final ArrayList<String> bundleIdsAdmin = new ArrayList<>();
+        final String systemUserId = adminUser.getUserId();
+        for(int i=1;i<5;i++){
+            bundleIdsAdmin.add(insertPublishingBundle(systemUserId, DateUtil.addDate(new Date(),
+                    Calendar.MONTH,-i)));
+        }
+        //Create a few bundles that the owner is a new User
+        final User newUser = new UserDataGen().nextPersisted();
+        final String newUserId = newUser.getUserId();
+        for(int i=1;i<5;i++){
+            bundleIdsAdmin.add(insertPublishingBundle(newUserId, DateUtil.addDate(new Date(),
+                    Calendar.MONTH,-i)));
+        }
+
+        //Create a bundle with today date
+        final String bundleTodayDate = insertPublishingBundle(systemUserId,new Date());
+
+        List<Bundle> bundlesSent = bundleFactory.findSentBundles(DateUtil.addDate(new Date(),
+                Calendar.DAY_OF_MONTH,-1),100,0);
+        List<String> bundlesSentIds = bundlesSent.stream().map(Bundle::getId).collect(Collectors.toList());
+
+        //All bundles on the list should be returned since the Admin can see all the bundles
+        for(final String bundleId : bundleIdsAdmin){
+            assertTrue(bundlesSentIds.contains(bundleId));
+        }
+        //Bundle with todays date should not be returned
+        assertFalse(bundlesSentIds.contains(bundleTodayDate));
+
+        bundleIdsAdmin.add(bundleTodayDate);
+        deletePublishingBundle(bundleIdsAdmin);
+
+    }
+
+    @Test
+    public void test_findSentBundlesOlderThan_byLimitedUser() throws DotDataException {
+        //Create a few bundles that the owner is the Admin
+        final ArrayList<String> bundleIdsAdmin = new ArrayList<>();
+        final String systemUserId = adminUser.getUserId();
+        for(int i=1;i<5;i++){
+            bundleIdsAdmin.add(insertPublishingBundle(systemUserId, DateUtil.addDate(new Date(),
+                    Calendar.MONTH,-i)));
+        }
+        //Create a few bundles that the owner is a new User
+        final ArrayList<String> bundleIdsUser = new ArrayList<>();
+        final User newUser = new UserDataGen().nextPersisted();
+        final String newUserId = newUser.getUserId();
+        for(int i=1;i<5;i++){
+            bundleIdsUser.add(insertPublishingBundle(newUserId, DateUtil.addDate(new Date(),
+                    Calendar.MONTH,-i)));
+        }
+
+        //Create a bundle with today date
+        final String bundleTodayDate = insertPublishingBundle(newUserId,new Date());
+
+        List<Bundle> bundlesSent = bundleFactory.findSentBundles(DateUtil.addDate(new Date(),
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Local variable 'bundlesSent' could be declared final](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)